bin/start-collectd

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+: ${GRAPHITE_HOST:=}
+: ${GRAPHITE_PORT:="2003"}
+
+set -eu
+
+CONF_TEMPLATE=/etc/collectd/collectd.conf.tpl
+CONF_DEST=/etc/collectd/collectd.conf
+
+HOSTNAME=$(wget -O - --tries 1 --timeout 0.2 http://169.254.169.254/1.0/meta-data/instance-id 2>/dev/null || hostname)
+
+if [ -n "$GRAPHITE_HOST" ]; then
+    sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+        -e "s/{{ graphite_host }}/$GRAPHITE_HOST/g" \
+        -e "s/{{ graphite_port }}/$GRAPHITE_PORT/g" "$CONF_TEMPLATE"
+else
+  sed -e "s/{{ hostname }}/$HOSTNAME/g" \
+      -e "s/LoadPlugin write_graphite/LoadPlugin write_log/g" \
```

Comment:
Ok, that makes sense, but I think there might be a better option here, namely to just have this bootstrap script print a notice like `INFO: GRAPHITE_HOST not provided so collectd will not be started` and then just exit with a success returncode. In supervisor's default configuration it will only autorestart processes that exit with a nonzero status code, so this should work just fine.

After all, if we're not sending the metrics anywhere we might as well just not run collectd.